src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -870,8 +880,6 @@ private void InitializeHelix()
             {
                 Model3DDictionary.Add(DefaultAxesName, axesModel3D);
             }
-
-            AttachAllGeometryModel3DToRenderHost();
```

Comment:
attaching here does not make any sense because it triggers `RequestAttachToScene` event but there are no listeners for this yet